elections/steering/2024/README.md

```diff
@@ -85,6 +85,8 @@ Corporate affiliation is applied after the election. If an organization finds
 itself with too many representatives it is up to those individuals to come
 to a consensus on who should serve on the committee.
 
+Note: to quickly check if you are an eligible voter you can look your GitHub username up in [this yaml file](https://github.com/kubernetes/community/blob/master/elections/steering/2024/voters.yaml).
```

Comment:
This is already mentioned in https://github.com/kubernetes/community/blob/532bbc690d0b7a6ff65a1f547549f49df7fea133/elections/steering/2024/README.md?plain=1#L227-L230.

Reply:
The election app requires logging in and navigating to the year prior to the election process (when I assume it's mandatory to log in if you want to cast a vote) while that file I've pointed to is generated and accessible simply via the repo, I think it is arguably faster but I'm very open to discuss.

Reply:
I don't understand why you think this addition is necessary?  Both methods of checking are already explained elsewhere in the same file.

Reply:
I think you've pointed out a legitmate voter documentation bug.  I don't think this is the way to solve it, though.  What would work better is to take the existing paragraph under Voting Process, and move it to Eligibility section, and then provide a back-link from Voting Process to the new location.  Do you follow?

Reply:
100% agreed upon, as I proposed in the previous comment.
Please take a look at the updated version!

Reply:
@jberkus did you have the chance to take a look?